INSTALL

```diff
@@ -1,5 +1,31 @@
-Installing Iris currently requires you to build it from source. The 
-latest release is available from https://github.com/SciTools/iris.
+You can either install Iris using the conda package manager or from source.
+
+Installing from conda
+---------------------
+
+Iris can be installed using the `conda <http://conda.pydata.org/>`_ package manager.
```

Comment:
Seems odd to have markup in plain text file? Most of the markup was removed [many moons ago](https://github.com/SciTools/iris/commit/9bdb292d84c8bab5b48168f47baac9b21a83fcb0#diff-1) so probably this should just be plain text too. (There are a few other bits of markup remaining which it would also be nice to remove, but that could be left to a separate PR.)